heka/_common.sls

```diff
@@ -0,0 +1,29 @@
+{%- from "heka/map.jinja" import server with context %}
+
+heka_packages:
+  pkg.latest:
+  - names: {{ server.pkgs }}
+
+/usr/share/lma_collector:
```

Comment:
Such custom things belong to `/usr/local` prefix, so `/usr/local/share/lma_collector`